Sources/SwifterSwift/CryptoKit/DigestExtensions.swift

```diff
@@ -0,0 +1,17 @@
+// DigestExtensions.swift - Copyright 2022 SwifterSwift
+
+#if canImport(CoreLocation)
```

Comment:
Wouldn't that be CriptoKit?
```suggestion
#if canImport(CryptoKit)
```

Reply:
Thanks @LucianoPAlmeida , good catch!